react/src/components/atoms/forms/InputCurrency/index.js

```diff
@@ -68,10 +68,11 @@ const Currency = (props) => (
               update.showError = false;
               update.errorMsg = errorMsg;
             }
-            context.updateState(update);
-            if (typeof props.onChange === 'function') {
-              props.onChange(e, numberValue);
-            }
+            context.updateState(update, () => {
+              if (typeof props.onChange === 'function') {
+                props.onChange(e, numberValue);
```

Comment:
should we also pass the props.id to the callback here `props.onChange(e, numberValue, props.id)`?